CONDUCT.md

```diff
@@ -54,7 +54,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
```

Comment:
Given that the repository is hosted within The Carpentries, its Code of Conduct and enforcement policy applies.
Please rename the file `CODE_OF_CONDUCT.md` (following GitHub's new recommendation) and use the content of this file: https://raw.githubusercontent.com/carpentries/.github/master/CODE_OF_CONDUCT.md Thanks!